plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieTableOperations.java

```diff
@@ -144,8 +158,9 @@ protected void commitMaterializedViewRefresh(TableMetadata base, TableMetadata m
         throw new UnsupportedOperationException();
     }
 
-    private static ContentKey toKey(SchemaTableName tableName)
+    private static ContentKey toKey(String databaseName, String tableName)
     {
-        return ContentKey.of(Namespace.parse(tableName.getSchemaName()), tableName.getTableName());
+        SchemaTableName schemaTableName = new SchemaTableName(databaseName, tableName);
+        return ContentKey.of(Namespace.parse(schemaTableName.getSchemaName()), schemaTableName.getTableName());
```

Comment:
Why not pass databaseName & tableName to `Namespace.parse()` directly?

Reply:
we miss the validation of empty schemaname and conversion of the identifiers into lower case.  
I don't know the background. But I assumed it was written for that purpose. So, retained the `SchemaTableName`